src/js/components/Button/README.md

```diff
@@ -163,10 +163,16 @@ function
 
 **badge**
 
-An indicator to show on the top-right of the Button. For accessibility,
-         supplement the badge by adding an a11yTitle to the Button which 
-         provides a useful message to screen readers. For example, "2 unread 
-         notifications".
+An indicator to show on the top-right of the Button. When badge is a 
+         boolean, the badge will be a circle with background color 'brand'. 
+         When badge is a number, that number will appear inside the badge. When 
+         badge is an object, background refers to the background color of the 
+         badge, value refers to the number that will appear inside the badge, 
```

Comment:
Are we assuming the value must be a number?

Reply:
Updating to include description that value can be a boolean, like the doc type shows